Wkhtmltopdf.cfc

```diff
@@ -123,7 +123,9 @@ component {
   private function _quote(required string val) {
     // escape and quote the value if it is a string and this isn't windows
     if (server.os.name != 'UNIX') {
-      val = reReplace(val, '(["\\$`])', '\\\1', 'all');
+      if(val CONTAINS " "){
```

Comment:
Thanks for your complete explanation :) Your modification looks good. To be honest I don't think we tested properly the escape regex :)
So you are using this snippet ?
```
val = replace(val, '"', '\"', 'all');
if(val CONTAINS " "){
   return '"#val#"';
}
```

If yes, could you please update the PR ?
Maybe we could modify `test.cfm` for applying such arguments.

Thanks you for your PR, it's really interesting to hear that people are using this module! And still using CF... :P